DEPS.bzl

```diff
@@ -3840,6 +3840,14 @@ def go_deps():
         sum = "h1:KBNDSne4vP5mbSWnJbO+51IMOXJB67QiYCSBrubbPRg=",
         version = "v1.2.2",
     )
+    go_repository(
+        name = "com_github_zabawaba99_go_gitignore",
+        build_file_proto_mode = "disable",
+        importpath = "github.com/zabawaba99/go-gitignore",
```

Comment:
Should not using the personal mod with few stars and not active recently!

Reply:
where it's imported in go codes, not found.

Reply:
Used here for the testmarker [config](https://github.com/pingcap/tidb/blob/33fc5944a95a91b79c8652120bbed47e92d28540/tests/testmarker/.testmarker.yaml) : https://github.com/pingcap/tidb/blob/33fc5944a95a91b79c8652120bbed47e92d28540/tests/testmarker/walker.go#L34

Reply:
3d47ffb607a8592e96c28a92542a542d5843a0ff resolves it and uses the nogo config style.